src/plugins/components/form/form-elements.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Form elements styles
+ */
+module.exports = function () {
+  return function ({ addComponents, theme }) {
+    const components = {
+      ".radio, .checkbox": {
+        backgroundColor: theme("colors.white"),
+        borderWidth: "3px",
+        borderColor: theme("colors.digital-blue.DEFAULT"),
+        width: "2.4rem",
+        height: "2.4rem",
+      },
+
+      ".radio:checked, .checkbox:checked": {
+        backgroundColor: theme("colors.digital-blue.DEFAULT"),
+      },
+
+      ".fieldset": {
+        border: "0",
+      },
+
+      ".label": {
+        display: "block",
+        maxWidth: theme("decanter.base.inputMaxWidth"),
+        fontSize: theme("decanter.base.inputFontSize"),
+      },
+
+      ".legend": {
+        ...theme("decanter.typography.typeB"),
```

Comment:
```suggestion
```
This was my mistake - we don't actually have a typeB typography style. However, it looks like we don't even have a Figma style for legend, so please just take this line out for now.